.circleci/config.yml

```diff
@@ -157,6 +158,21 @@ jobs:
            _build/debug/velox/examples/velox_example_expression_eval
            _build/debug/velox/examples/velox_example_opaque_type
 
+  linux-benchmarks:
+    executor: build
+    steps:
+      - checkout
+      - run:
+          name: "Building and running baseline (main)"
+          command: |
+            #git checkout HEAD^1 
```

Comment:
The idea here is to first go up one commit to compile the current state on "main" (without the current revision being tested). Hopefully this will mostly hit ccache.

We can't do it for now because main doesn't contains the new makefile target and this will fail :)